src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -23,7 +23,7 @@
 
 
     private KeyBindingCategory(String name) {
-        this.name = name;
+        this.name = name.replace("&", "");
```

Comment:
Well it's not really coincidence that the categories match the menu titles as I used the menu titles where the shortkey's action is located as the its category.